Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt

```diff
@@ -182,39 +173,23 @@ private fun ExploreItemRow(
 
 @Composable
 private fun ExploreImage(item: ExploreModel) {
-    Box {
-        val painter = rememberAsyncImagePainter(
-            model = ImageRequest.Builder(LocalContext.current)
-                .data(item.imageUrl)
-                .crossfade(true)
-                .build()
-        )
-
-        if (painter.state is Loading) {
```

Comment:
Why did you remove this loading state?

Reply:
It didn't look good when switching to different sized images, having nothing instead looks better than the jump in size change.